cudapoa/tests/Test_CudapoaAddAlignment.cu

```diff
@@ -41,6 +41,7 @@ public:
                    std::vector<uint8_t> read, std::vector<int8_t> base_weights, std::vector<int16_t> alignment_graph, std::vector<int16_t> alignment_read)
         : graph(nodes, outgoing_edges, node_alignments, node_coverage_counts)
         , read_(read)
+        , base_weights_(base_weights)
```

Comment:
I assume this is ok, but not sure

Reply:
this was throwing an `unused parameter` error - but the API of the class exposing accessing base_weights (though I assume it's not used anywhere yet).